cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -123,7 +125,7 @@ static CompositeFuture run(Vertx vertx, KubernetesClient client, PlatformFeature
                             "ABCDEFGHIJKLMNOPQRSTUVWXYZ" +
                             "0123456789");
 
-            kafkaClusterOperations = new KafkaAssemblyOperator(vertx, pfa, certManager, passwordGenerator, resourceOperatorSupplier, config);
+            kafkaClusterOperations = new KafkaAssemblyOperator(vertx, pfa, certManager, passwordGenerator, resourceOperatorSupplier, config, eventsPublisher);
```

Comment:
Any chance we can get this passed through the `ResourceOperatorSupplier`? I think it would make many things a lot easier -> mocking, all tests, propagation etc.

Reply:
Will do :)

Reply:
@scholzj I just pushed a change where the event publisher is provided via ROS :)